src/api/submission/validations2/validators/classification_benthic_photo_quadrat.py

```diff
@@ -51,31 +54,40 @@ def __call__(self, *args, **kwargs):
                     Image.objects.filter(collect_record_id=image.collect_record_id)
                 )
             }
+
+        context = {
+            "image_id": image_id,
+            "quadrat_num": self._cache_quadrat_num_lookup[image_id],
+            "missing_num_annotations": None,
+        }
+
         annos = Annotation.objects.select_related("point", "point__image").filter(
             is_confirmed=True, point__image_id=image_id
         )
 
-        classifier = annos[0].classifier
+        classifier = Classifier.objects.get_or_none(id=collect_record.get("classifier_id"))
+        if not classifier:
+            classifier = Classifier.latest()
+            if not classifier:
+                return OK, None, [context]
```

Comment:
Why is `context` a list? That looks inconsistent with what's getting returned elsewhere.